service/routes.py

```diff
@@ -168,6 +195,15 @@ def check_content_type(content_type):
     )
 
 
+
+######################################################################
+# Logs error messages before aborting
+######################################################################
+def error(status_code, reason):
```

Comment:
can you move this entire block to the end of the file and add `UTILITIES` instead of the `Logs error mesages...`? just following the same convention that we have in other files

Reply:
Done!